src/iris/api.py

```diff
@@ -1179,6 +1187,7 @@ def on_get(self, req, resp, plan_id):
 
             cursor.execute(single_plan_query_tags, plan['id'])
             plan['tags'] = cursor.fetchall()
+            plan['dynamic_tracking'] = bool(plan.get('dynamic_tracking'))
```

Comment:
Nit: could we use [] instead of get() here, as the field is guaranteed to be in the response as it is in the select query?

Reply:
changed